include/bios_disk.h

```diff
@@ -77,8 +81,8 @@ void incrementFDD(void);
 
 #define MAX_DISK_IMAGES (2 + MAX_HDD_IMAGES)
 
-extern imageDisk *imageDiskList[MAX_DISK_IMAGES];
-extern imageDisk *diskSwap[MAX_SWAPPABLE_DISKS];
+extern std::vector<std::unique_ptr<imageDisk>> imageDiskList;
+extern std::vector<std::unique_ptr<imageDisk>> diskSwap;
```

Comment:
`std::array` would be closer to the original code while still giving all benefits of C++11 containers.

Reply:
Ahh right; these two array sizes are locked in and I initialize them to their permanent size; will make this change.

Reply:
I missed this comment in the original list; I guess I don't see when GitHub hides comments.. Pushing an update for the arrays.  Tested and working fine.